### ISSUE TITLE
Explain explicitly how to re-enable help text for "search hit BOTTOM, continuing at TOP"

### ISSUE THREAD

**Issue by kyhy** [wincent/loupe#5]
is there a way to show status "search hit BOTTOM, continuing at TOP" when search loops back to the top?

**Comment by wincent** [comment]
That should happen automatically; see `:h loupe-shortmess-override`. Basically:

```
set shm+=s
```

Maybe you have something else in your set-up that is setting `'shortmessage'`? Let me know.

**Comment by wincent** [comment]
Oh, sorry, I read the exact opposite of what you actually said. In your case you _want_ the message, so you would set:

```
set shm-=s
```

In `~/.vim/after/plugin/loupe.vim`.

Let me know if that doesn't work for you.

**Comment by kyhy** [comment]
Still not seeing the message :[ 

Here is my [vimrc](https://github.com/kyhy/dotfiles/tree/master/vim)

**Comment by wincent** [comment]
Not at my computer now, because it appears busted (won't turn on!) but your files look right to me.

**Comment by wincent** [comment]
Just tested it, `set shm-=s` in your `~/.vim/after/plugin/loupe.vim` works for sure.

If it's not working for you, there are two things you can try:
1. Inspect `:scriptnames` to confirm that `~/.vim/after/plugin/loupe.vim` _is_ getting evaluated, and getting evaluated _after_ the `loupe.vim` file inside Loupe itself.
2. Run `:verbose set shm?` to see where `'shm'` was last set.

**Comment by wincent** [comment]
I've updated the docs to hopefully make this clearer. The was already info in there on how to do this, but it wasn't very obvious, so I added [a heading](https://github.com/wincent/loupe/tree/e1d7a6972c39dcff763c271c7125ccedf3ee0c99#preventing-loupe-overrides-from-taking-effect) and some other copy in commit e1d7a6972c39dcf.

Going to close this as I don't think there's anything else actionable to do here, but feel free to comment again if you have anything to add/ask in relation to your troubleshooting.
